CHANGELOG.rst

```diff
@@ -28,6 +28,10 @@ number of the code change for that issue.  These PRs can be viewed at:
 
 3.7.1 (unreleased)
 ==================
+- Added an if statement in verify_crthresh() so the evaluation for the possible
+  rejection of catalogs due to not enough sources vs estimated cosmic rays is
+  not done for ACS/SBC and WFC3/IR. [#nnnn]
```

Comment:
Could be clearer, I think. Also replace `[#nnnn]` with the actual PR number.

Reply:
DONE